src/core/demangle.d

```diff
@@ -207,8 +207,9 @@ pure @safe:
         if( val.length )
         {
             assert( contains( dst[0 .. len], val ) );
+            if (len + val.length > dst.length)
+                overflow();
```

Comment:
Why would removing something have to cause an overflow?

Reply:
updated